src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -110,7 +112,7 @@ public Claim requestMoreTimeToRespond(@PathVariable("claimId") Long claimId,
     @GetMapping("/{caseReference}/defendant-link-status")
     @ApiOperation("Check whether a claim is linked to a defendant")
     public DefendantLinkStatus isDefendantLinked(@PathVariable("caseReference") String caseReference) {
-        Boolean linked = claimService.getClaimByReference(caseReference)
+        Boolean linked = claimService.getClaimByReferenceAnonymous(caseReference)
```

Comment:
I would like to delete this endpoint in the future as it provides minimal value. It does not impact this PR of course.

Reply:
needs idam to be fixed for that to happen, now that there's developers on it again we can potentially do that